frontend/src/metabase/public/components/widgets/SharingPane/icons/PublicEmbedIcon/PublicEmbedIcon.styled.tsx

```diff
@@ -0,0 +1,3 @@
+import styled from "@emotion/styled";
+
+export const PublicEmbedIconWrapper = styled.svg``;
```

Comment:
If there's a better way to ensure that the hover state works for both of these icons beyond what I've done, I would appreciate the feedback here. Not sure what else I can do

Reply:
You can inline `svg` where you use `PublicEmbedIconWrapper` and make `PublicEmbedIcon` accepts `className` then pass it to the `svg` tag.

Reply:
Sweet, let me know if I did what you were thinking